<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QMainWindow" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSslChat Server</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="serverGroupBox">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="portLabel">
         <property name="text">
          <string>Server Port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="portSpinBox">
         <property name="minimum">
          <number>1024</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>32767</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="startStopButon">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Start Server</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="sslFilesGroupBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="keyLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Key</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="certificateLabel">
         <property name="text">
          <string>Certificate</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="keyLineEdit"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="certificateLineEdit"/>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="keyButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="certificateButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="logTextEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
